eureka-core/src/main/java/com/netflix/eureka/util/EurekaMonitors.java

```diff
@@ -109,13 +114,26 @@ public void increment() {
      *            true if this a replication, false otherwise.
      */
     public void increment(boolean isReplication) {
-        counter.incrementAndGet();
+        incrementCounter();
 
         if (!isReplication) {
-            myZoneCounter.incrementAndGet();
+            incrementMyZoneCounter();
+        }
```

Comment:
Typically we recommend modeling it so you can drill into the data easier. I would suggest making whether or not it is for replication a dimension, for example:

```java
private final Counter replicationCounter =
    Spectator.globalRegistry().counter("eureka." + name, "isReplication", "true");
private final Counter myZoneCounter =
    Spectator.globalRegistry().counter("eureka." + name, "isReplication", "false");

public void increment(boolean isReplication) {
  Counter c = isReplication ? replicationCounter : myZoneCounter;
  c.increment();
}
```

When querying the old `count` version would just be just restrict to a given name and it will automatically sum across the other dimensions. You could further restrict based on `isReplication` to get the part where that is true or false. This also makes it easy to group by `isReplication` to see both or visualize a percentage.